docs/presentation/Content/ContentInstancesOfSpecificClasses.md

```diff
@@ -2,38 +2,161 @@
+
+### Attribute: `priority`
+
+> **Default value:** `1000`
+
+Defines the order in which specifications are handled - higher priority means the specifications is handled first. If priorities are equal, the specifications are handled in the order they're defined.
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.Priority.Ruleset]]
+```
+
+![Example of using priority attribute](./media/contentInstancesOfSpecificClasses-with-priority-attribute.png)
+
+### Attribute: `relatedProperties`
+
+Specifications of [related properties](./Terminology.md#related-properties) which are included in the generated content. *See [this page](./RelatedPropertiesSpecification.md) for more details.*
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.RelatedProperties.Ruleset]]
+```
+
+  |  | Result                                                                                                                                      |
+  | ------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+  | without  | ![Example when doing normal property select](./media/contentInstancesOfSpecificClasses-with-relatedProperties-attribute-1.png) |
+  | with    | ![Example when selecting with related properties](./media/contentInstancesOfSpecificClasses-with-relatedProperties-attribute-2.png)     |
+
+### Attribute: `calculatedProperties`
+
+Specifications of calculated properties whose values are generated using provided ECExpressions. *See [this page](./CalculatedPropertiesSpecification.md) for more details.*
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.CalculatedProperties.Ruleset]]
+```
+
+![Example of using calculatedProperties attribute](./media/contentInstancesOfSpecificClasses-with-calculatedProperties-attribute.png)
+
+### Attribute: `propertyCategories`
+
+Specifications for custom categories. Simply defining the categories does nothing - they have to be referenced from `PropertySpecification` defined in `propertyOverrides` by `id`. *See [this page](./PropertyCategorySpecification.md) for more details.*
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.PropertyCategories.Ruleset]]
+```
+
+![Example of using propertyCategories attribute](./media/contentInstancesOfSpecificClasses-with-propertyCategories-attribute.png)
+
+### Attribute: `propertyOverrides`
+
+Specifications for various property overrides. *See [this page](./PropertySpecification.md) for more details.*
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.PropertyOverrides.Ruleset]]
+```
+
+![Example of using propertyOverrides attribute](./media/contentInstancesOfSpecificClasses-with-propertyOverrides-attribute.png)
+
+### Attribute: `showImages`
+
+> **Default value:** `false`
+
+Should image IDs be calculated for the returned instances. When `true`, [ImageIdOverride](../customization/ImageIdOverride.md) rules get applied when creating.
```

Comment:
```suggestion
Should image IDs be calculated for the returned instances. When `true`, [ImageIdOverride](../customization/ImageIdOverride.md) rules get applied when creating the content.
```

Also, would be nice to add a test with a snippet similar to [this](https://github.com/iTwin/itwinjs-core/blob/ea2f09b23710401e2f37a3c25af71b5e459304bd/full-stack-tests/presentation/src/learning-snippets/Hierarchies.test.ts#L1036-L1042).

Reply:
No change. Planned removal.

Reply:
You're right, I completely forgot we were thinking of removing the `showImages` attribute, so there's no point to add a code snippet for it.

However, my suggested change for the description still applies.